src/app.js

```diff
@@ -889,6 +890,47 @@ function run () {
   var node = document.getElementById('staticanalysisView')
   node.insertBefore(staticanalysis.render(), node.childNodes[0])
 
+  // ----------------- Tx listener -----------------
+
+  var txlistener = new Txlistener({
+    appAPI: {
```

Comment:
For all modules I started to call it just `api` or `events`, like:
```js
var txlistener = new Txlistener({
  api: { /* .... */ },
  events: { app: self.event, udapp: udap.event /* , .... */ }
})
```
To me this looks cleaner and communicates the same message :-)